pandas/io/parsers.py

```diff
@@ -579,6 +579,7 @@ def read_csv(
     compression="infer",
     thousands=None,
     decimal: str = ".",
+    line_terminator=None,
```

Comment:
again, in the signature, I think we should only have the one parameter and the compatibility keyword accepted though **kwargs